lib/cqm/converter/hds_record.rb

```diff
@@ -108,6 +98,22 @@ def to_qdm(record)
       patient.family_name = record.last if record.last
       patient.bundle_id = record.bundle_id if record.bundle_id
 
+      # Convert extended_data.
+      patient.extended_data = {}
+      patient.extended_data['type'] = record.type if record.respond_to?('type')
+      patient.extended_data['measure_ids'] = record.measure_ids if record.respond_to?('measure_ids')
+      patient.extended_data['source_data_criteria'] = record.source_data_criteria if record.respond_to?('source_data_criteria')
+      patient.extended_data['expected_values'] = record.expected_values if record.respond_to?('expected_values')
```

Comment:
This line may need to be modified based on the final design of the new "Expected Values" work, because Expected_values will most likely exist on the patient model and not in the extended_data.

Reply:
i'm thinking along the lines of @holmesie